src/resolve-template.js

```diff
@@ -0,0 +1,90 @@
+var path = require("path");
+var fileSystem = require('fs');
+var readdir = require('recursive-readdir');
+var ContextElementDependency = require('webpack/lib/dependencies/ContextElementDependency');
+var assign = Object.assign || require('object.assign');
+
+var cheerio = require('cheerio');
+
+/**
+ * Generates key-value pairs or <require from="paths"> and their corresponding files.
+ */
```

Comment:
@EisenbergEffect @atsu85 Indeed it is possible to invent new ones, plus it is possible to generate paths dynamically. Something like this cannot be supported by statically analyzing files. For that purpose I'm implementing an additional way to declare external dependencies and for Aurelia plugins to declare internal dependencies by listing those additional resources in the `package.json` file. You'd add something like:

```
{
   ...
   "aurelia": {
      "resources": [ 'dist/some-resource.js', 'dist/another.html', 'dist/another-without-extension' ]
    }
}
```
And the Webpack plugin will read that and know to include those files in the bundle.